lib/src/core/filters.dart

```diff
@@ -165,7 +126,7 @@ class FilterEvent {
       this.topics});
 
   FilterEvent.fromMap(Map<String, dynamic> log)
-      : removed = log['removed'] as bool ?? false,
+      : removed = log['removed'] as bool,
```

Comment:
The idea was to support maps without a `removed` key, can we do

```suggestion
      : removed = log['removed'] as bool? ?? false,
```